documentation/import-from-rdbms.adoc

```diff
@@ -0,0 +1,120 @@
+
+- Direct connection  - here you will be providing the import service with access to your data source including  connection url, database name and credentials. 
+- Proivision of CSV files - here you will be providing CSV files from your filesystem as the data source. A separate guide "Import from CSV" is available to take you though this process.
+
+This guide will focus on connecting directly to a data source. Start by clicking the "New data source" button on the Data Sources tab, which will present you with a number of data source options. In this guide we'll connect to PostgreSQL.
+
+
+Complete the form with the required information for your data source. If there are any problems connecting to your data source, you'll stay on the connection form and have the opportunity to tweak the details before continuing.
+
+image::import-source-config-form.png[]
+
+[TIP]
+====
+Currently error reporting is limited, so you might find it helpful to validate all your details are correct by successfully connecting from your development machine. It's worth re-iterating that at present your database needs to be publicly accessible for us to conenct to it.
+====
+
+Once the connection is successful, you'll be prompted to review and confirm the table schema for the database you've chosen to connect to. Here you can see a summary of the tables and columns in your database to help validate you're connected to the right data source. If everything looks good, hit `Confirm`. This will add your data source and and save it for future re-use.
+
+image::import-source-table-schema.png[]
+
+[TIP]
+====
+While retain your connection details, we don't currently retain your password. You'll need to re-provide the password for your data source when you use it again or trigger the import job.
+====
+
+Once you've added a data source, you can either start generating a candidate graph from it, or hit `Later` to come back and do this at a later point in time. If you come back later, you'll be able to use the `...` button on a row and the `New graph model from data source` option to start the graph generation flow. In this guide, let's move immediately onto the next stage of generating a graph model.
+
+
+== Generating a graph model
+
+
+After adding a data source, hit the `Generate graph model` button. Doing this inspects any constraints that exist in your database, looking for primary key constraints to create nodes with IDs and foreign keys to generate relationships between them. If suitable contraints are found we'll generate a corresponding candidate graph for and map the tables and columns to it. If your dataset doesn't have any such constraints (or only part of the schema does), you'll have to map your tables to nodes and relationships manually. If you're not familiar with your data source, it might take some time to understand the different entities and connections present in your tables.
```

Comment:
```suggestion
Use the `Generate graph model` button once you have added a data source. 
Doing this inspects any constraints that exist in your database, looking for primary key constraints to create nodes with IDs and foreign keys to generate relationships between them. 
If suitable constraints are found, a corresponding candidate graph is generated, and the tables and columns are mapped to it. 
If your dataset doesn't have any such constraints (or only part of the schema does), you have to map your tables to nodes and relationships manually. 
If you're not familiar with your data source, it may take some time to understand the different entities and connections present in your tables.
```